### PR TITLE
chore(CI): Improve labeler

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  
-  

## Issues Addressed:
- Closes 

## Tests Performed:
-
-

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: .github/labeler-review.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Ready to be reviewed:
+- '**/*'
+
+Waiting to be tested:
+- '**/*'
```
Comment (FrancescoBorzi) [inline .github/labeler-review.yml:5]: not sure about this, the PR might not be ready when it's opened

Comment (Kitzunu) [reply .github/labeler-review.yml:5]: Check labeler.yml or whatever the name is.

If it is in draft it wont apply

Comment (Kitzunu) [reply .github/labeler-review.yml:5]: runs-on: ubuntu-20.04
    if: (github.event.pull_request.draft == false) || (github.event.action.synchronize == false) || (github.event.action.unlabled == false) && (github.event.action.ready_for_review == true)
    steps:

Comment (FrancescoBorzi) [reply .github/labeler-review.yml:5]: ok, is there a way to also filter for PRs that have passed the build?

Comment (Kitzunu) [reply .github/labeler-review.yml:5]: I can check in the eeek

Comment (Kitzunu) [reply .github/labeler-review.yml:5]: I think not.
But that part will only trigger if
Not a Draft PR
Not being rebased/synchronized with Master (Using the button below a PR)
Not if unlabled
and if it is ready for review (Not in draft, don't know why they have two different hooks for this)
@FrancescoBorzi

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request closed without merging**